metronome/config/specs/src/io/iohk/metronome/config/ConfigParserSpec.scala

```diff
@@ -53,6 +53,32 @@ class ConfigParserSpec
     json.noSpaces shouldBe """{"nestedStructure":{"barBaz":{"spam":"eggs"},"foo":10}}"""
   }
 
+  "seq,list,vector decoders" should "handle JSON Object with indices successfully" in {
+    import ConfigParserSpec.TestConfigWithJsonArray._
+    val withArray = """{"field":["valueA", "valueB", "valueC"]}"""
+    val withObject = """{"field":{"2":"valueC", "0": "valueA", "1":"valueB"}}"""
+    val aConf = ConfigFactory.parseString(withArray)
+    val oConf = ConfigFactory.parseString(withObject)
+
+    //seqs
+    val aSeq = ConfigParser.parse[TestConfigWithSeq](aConf.root)
+    val oSeq = ConfigParser.parse[TestConfigWithSeq](oConf.root)
+    aSeq shouldBe a[Right[_,_]]
```

Comment:
It would be good to actually check that the parsed result contains what we expect. Could be just an empty list, and the test wouldn't highlight it.

Reply:
That's done!